applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -107,17 +108,29 @@ static void advertising_process(struct k_work *work)
 	} else
 #endif
 	{
-		adv_param = *BT_LE_ADV_CONN;
-		adv_param.options |= BT_LE_ADV_OPT_ONE_TIME;
+		/* Create a non-connectable non-scannable advertising set */
+		ret = bt_le_ext_adv_create(BT_LE_EXT_ADV_CONN_NAME, NULL, &adv);
+
+		if (ret) {
+			LOG_ERR("Failed to create advertising set: %d", ret);
+			return;
+		}
+
+		ret = bt_le_ext_adv_set_data(adv, ad_peer, ARRAY_SIZE(ad_peer), NULL, 0);
+
+		if (ret) {
+			LOG_ERR("Failed to set advertising data: %d", ret);
+			return;
+		}
 
-		ret = bt_le_adv_start(&adv_param, ad_peer, ARRAY_SIZE(ad_peer), NULL, 0);
+		ret = bt_le_ext_adv_start(adv, BT_LE_EXT_ADV_START_DEFAULT);
 
 		if (ret) {
-			LOG_ERR("Advertising failed to start (ret %d)", ret);
+			LOG_ERR("Failed to start advertising set: %d", ret);
```

Comment:
If `bt_le_ext_adv_start` returns an error the print will be "Failed to start advertising set: -X" which will look strange.